src/core_modules/capture-core/reducers/descriptions/dataEntry.reducerDescription.js

```diff
@@ -266,6 +219,7 @@ export const dataEntriesFieldsUIDesc = createReducerDescription({
         dataEntryValuesUI[payload.fieldId] = { ...dataEntryValuesUI[payload.fieldId], ...payload.valueMeta, modified: true };
         return newState;
     },
+    [registrationFormActionTypes.CLEAN_UP_DATA_ENTRY]: cleanUp,
```

Comment:
This action type is not defined. Hence, could you reflect on whether this cleanup is really needed?

Reply:
Hey that was a mistake of mine. I wanted to clean the data entry when the user is leaving the `newEnrollment` page. Is should be fixed now.